src/components/Slider/scss/Slider.scss

```diff
@@ -0,0 +1,118 @@
+.container {
+  width: 100%;
+  margin: auto;
+  & > h2 {
+    font-size: 42px;
+    text-align: center;
+  }
+
+  & > p {
+    text-align: center;
+    margin-top: 40px !important;
```

Comment:
be careful with '!important' it may couse a lot of unwanted complications, use only if necessary